core/src/blocks/search.rs

```diff
@@ -230,7 +252,7 @@ impl Block for Search {
 
                 let body = json!({
                     "q": query,
-                    "num": self.num.unwrap_or(10),
+                    "num": num.unwrap_or(8),
```

Comment:
Why reducing it?

Reply:
We always specify it and I think the default of the other provider is 8 so to align :)